event/event.go

```diff
@@ -25,32 +25,30 @@ package event
 type Event interface {
 }
 
-// KeyCharacter is an event that occurs when a character is actually typed on
+// KeyboardKeyCharacter is an event that occurs when a character is actually typed on
 // the keyboard. This may be provided by an input method.
-type KeyCharacter struct {
-	// Code is the key code of the key typed.
-	// TODO: this should change later from an int to an enumeration type.
-	Code int
+type KeyboardKeyCharacter struct {
+	// Key is the key code of the key typed.
+	Key Key
 	// Modifiers are the modifiers pressed together with the key.
 	// TODO: this should change later from an int to an enumeration type.
 	Modifiers int
 	// Character is the character that was typed.
 	Character rune
 }
 
-// KeyDown is an event that occurs when a key is pressed on the keyboard.
-type KeyDown struct {
-	// Code is the key code of the key pressed or released.
-	// TODO: this should change later from an int to an enumeration type.
-	Code int
+// KeyboardKeyDown is an event that occurs when a key is pressed on the keyboard.
+type KeyboardKeyDown struct {
```

Comment:
The name KetboardKeyDown is longer, but consistent with names like MouseButtonDown, so I would say this is an improvement after all.